.github/actions/install-python-env/action.yml

```diff
@@ -7,10 +7,16 @@ inputs:
     description: The version of Python to use (passed to actions/setup-python)
     required: true
     default: "3.11"
+  fetch-depth:
+    description: The depth of the commit history to fetch during checkout
+    required: false
+    default: 1
```

Comment:
I know that you invoke this action with the value 0 from your workflow, but I wonder if it'd make more sense for this to default to 0, too, since your workflow relies on comparing pairwise commits through the entire history, not just the most recent commit.

Reply:
I was conflicted on which one to use as default, but I agree this makes more sense!

Reply:
Fair enough; I wasn't considering using the action outside of doing the entropy calculation, which AFAIK does require at least some commit history. If you have other uses for the action in mind, then sure, 1 works for me.